main.ml

```diff
@@ -1277,7 +1286,7 @@ let self_test_plugin ~root_dir plugin =
 let self_test ~root_dir =
   (self_test_plugin ~root_dir "org.xen.xapi.storage.dummy"
    >>>= fun () ->
-   self_test_plugin ~root_dir "org.xen.xapi.storage.dummyv4")
+   self_test_plugin ~root_dir "org.xen.xapi.storage.dummyv5")
```

Comment:
Do you want to add some self-tests for v5 datapath?

Reply:
It would be good to do later, but it's out of scope for this PR